qa/workunits/rbd/rbd_mirror.sh

```diff
@@ -465,7 +555,7 @@ request_resync_image ${CLUSTER1} ${POOL} ${image} image_id
 wait_for_image_present ${CLUSTER1} ${POOL} ${image} 'deleted' ${image_id}
 wait_for_image_present ${CLUSTER1} ${POOL} ${image} 'present'
 wait_for_image_replay_started ${CLUSTER1} ${POOL} ${image}
-wait_for_status_in_pool_dir ${CLUSTER1} ${POOL} ${image} 'up+replaying' 'primary_position'
+wait_for_status_in_pool_dir ${CLUSTER1} ${POOL} ${image} 'up+replaying' ${DESC_PATTERN}
 compare_images ${POOL} ${image}
 
 testlog "TEST: image resync while replayer is stopped"
```

Comment:
It would be better to pull this line into the `if` like in the original `rbd_mirror_snapshot.sh` since this test is executed conditionally.